### PR TITLE
Updates yo-app to v1.0.

### PR DESCRIPTION
**Pull request by joeldudley**
Key changes:
- Had to split transaction and flows tests up because of new package-setting stuff
- Had to un-nest YoState from YoContract to allow the state to be queried from the crash shell

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 group 'com.github.roger3cev'
-version '3'
+version '4'
 
 buildscript {
-    ext.corda_release_version = '0.13.0'
-    ext.corda_gradle_plugins_version = '0.11.2'
-    ext.kotlin_version = '1.1.2'
-    ext.junit_version = '4.12'
+    ext.corda_release_version = '1.0.0-RC3'
```
Comment (roger-that-dev) [inline build.gradle]: I think we can update this to V1.0.0 now.

File: build.gradle [hunk 2]
```diff
@@ -16,29 +16,25 @@ buildscript {
...
     mavenLocal()
-    mavenCentral()
     jcenter()
+    mavenCentral()
     maven { url 'https://dl.bintray.com/kotlin/exposed' }
     maven { url 'https://jitpack.io' }
+    maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases' }
 }
 
+apply plugin: 'java'
```
Comment (roger-that-dev) [inline build.gradle]: do we need the java plugin?

**Review by roger-that-dev — APPROVED** [review]
Two super minor things!

File: build.gradle (hunk 2 above)
Comment (joeldudley) [reply build.gradle]: No. I think the Kotlin plugin actually applies the Java plugin anyway.

**Pull request merged**